### PR TITLE
Add scoring model on test set

### PR DESCRIPTION
**Pull request by ArturoAmorQ**
In the same line of thought as in #439, after finding the best parameter on the train-validation sets in [Exercise M3.01](https://inria.github.io/scikit-learn-mooc/python_scripts/parameter_tuning_ex_02.html) and its [solution](https://inria.github.io/scikit-learn-mooc/python_scripts/parameter_tuning_sol_02.html), we should ask the student to score it on the test set. Else the role of the initial train-test split is unclear.

### CODE REVIEW

**Review by ogrisel — APPROVED** [review]
LGTM, just a few comments:
